### PR TITLE
Fixes issue where `service stop` does not work in `linux` systems. Closes #653

### LINKED ISSUE #653: implicit service stop times out in linux

### PR DESCRIPTION
**Pull request by binaek89**

### CODE REVIEW

File: db/stop_database.go [hunk 1]
```diff
@@ -188,16 +182,11 @@ func doThreeStepPostgresExit(process *psutils.Process) error {
...
-		// MURDER
-		err = killProcessTree(process)
-		if err != nil {
-			return err
-		}
-		exitSuccessful = waitForProcessExit(process, 5*time.Second)
-	}
 
 	if !exitSuccessful {
+		log.Printf("[ERROR] ** > Failed to stop service\n")
```
Comment (kaidaguerre) [inline db/stop_database.go:187]: do we want all this logging in production code?

Comment (kaidaguerre) [reply db/stop_database.go:187]: should it be at TRACE instead? so we can enable it as required?

Comment (binaek89) [reply db/stop_database.go:187]: This code should never be executed . If it comes here, then we will get this as a BIG RED FLAG!!!

Comment (kaidaguerre) [reply db/stop_database.go:187]: all the same, I don't think we want a wall of *ERROR* lines - we could print a single error line, and then format the remaining information in to an error report which is printed underneath

**Comment by binaek89** [comment]
When the service fails to stop, the following log is printed out:
```
[ERROR] Failed to stop service
[ERROR] Service Details:
 **  > Name: postgres
 **  > CmdLine: /Users/binaeksarkar/.steampipe/db/12.1.0/postgres/bin/postgres -p 9193 -c listen_addresses="*" -c application_name=steampipe -c cluster_name=steampipe -c autovacuum=off -c bgwriter_lru_maxpages=0 -c effective-cache-size=64kB -c fsync=off -c full_page_writes=off -c maintenance-work-mem=1024kB -c password_encryption=scram-sha-256 -c random-page-cost=0.01 -c seq-page-cost=0.01 -c temp-buffers=800kB -c timezone=UTC -c track_activities=off -c track_counts=off -c wal-buffers=32kB -c work-mem=64kB -c jit=off -c log_statement=all -c log_min_duration_statement=2000 -c logging_collector=on -c log_min_error_statement=error -c log_directory=/Users/binaeksarkar/.steampipe/logs -c log_filename=database-%Y-%m-%d.log -c ssl=on -c ssl_cert_file=/Users/binaeksarkar/.steampipe/db/12.1.0/data/server.crt -c ssl_key_file=/Users/binaeksarkar/.steampipe/db/12.1.0/data/server.key -D /Users/binaeksarkar/.steampipe/db/12.1.0/data
 **  > Status: S
 **  > Executable: /Users/binaeksarkar/.steampipe/db/12.1.0/postgres/bin/postgres
 **  > Username: binaeksarkar
 **  > Parent Details
 **    > Name: launchd
 **    > CmdLine: /sbin/launchd
 **    > Status: S
 **    > Executable: /sbin/launchd
 **    > Username: root
 **  > Children Details
 **    > Name: postgres
 **    > CmdLine: postgres: steampipe: logger
 **    > Status: S
 **    > Executable: /Users/binaeksarkar/.steampipe/db/12.1.0/postgres/bin/postgres
 **    > Username: binaeksarkar

 **    > Name: postgres
 **    > CmdLine: postgres: steampipe: checkpointer
 **    > Status: S
 **    > Executable: /Users/binaeksarkar/.steampipe/db/12.1.0/postgres/bin/postgres
 **    > Username: binaeksarkar

 **    > Name: postgres
 **    > CmdLine: postgres: steampipe: background writer
 **    > Status: S
 **    > Executable: /Users/binaeksarkar/.steampipe/db/12.1.0/postgres/bin/postgres
 **    > Username: binaeksarkar

 **    > Name: postgres
 **    > CmdLine: postgres: steampipe: walwriter
 **    > Status: S
 **    > Executable: /Users/binaeksarkar/.steampipe/db/12.1.0/postgres/bin/postgres
 **    > Username: binaeksarkar

 **    > Name: postgres
 **    > CmdLine: postgres: steampipe: stats collector
 **    > Status: S
 **    > Executable: /Users/binaeksarkar/.steampipe/db/12.1.0/postgres/bin/postgres
 **    > Username: binaeksarkar

 **    > Name: postgres
 **    > CmdLine: postgres: steampipe: logical replication launcher
 **    > Status: S
 **    > Executable: /Users/binaeksarkar/.steampipe/db/12.1.0/postgres/bin/postgres
 **    > Username: binaeksarkar
```

File: db/stop_database.go (hunk 1 above)
Comment (binaek89) [reply db/stop_database.go]: Updated

**Pull request merged**